src/bindings/python/python.cc

```diff
@@ -93,7 +130,8 @@ struct simplified_actor_t : public valhalla::tyr::actor_t {
 };
 
 PYBIND11_MODULE(python_valhalla, m) {
-  m.def("Configure", py_configure);
+  m.def("Configure", py_configure, py::arg("config_file"), py::arg("config") = py::dict(),
+        py::arg("tile_dir") = "", py::arg("tile_extract") = "", py::arg("verbose") = true);
```

Comment:
what this'll do on the python side:
- if there's config_file and no config was specified nothing happens
- the config has to be a full valhalla config, `valhalla.config.get_default()` will load the config dict, and it'll overwrite an existing config file
- tile_dir and tile_extract are just convenience so you don't have to specify them manually in the config dict